en/micro-integrator/docs/references/connectors/redis-connector/redis-connector-example.md

```diff
@@ -0,0 +1,453 @@
+    
+        <img src="../../../../assets/img/connectors/redis-init-parameterspng.png" title="Add values to the init operation" width="800" alt="Add values to the init operation"/>
+
+5. Set up the iPush operation. In this operation we can inserts one or more elements to the head of a list.
+
+    1. Navigate into the **Palette** pane and select the graphical operations icons listed under **Redis Connector** section. Then drag and drop the `iPush` operation into the Design pane.
+           
+        <img src="../../../../assets/img/connectors/redis-ipush-operation.png" title="Drag and drop ipush operation" width="500" alt="Drag and drop ipush operation"/>    
+
+    2. In this operation we are going to inserts one or more elements to the head of a list.The lPush operation inserts one or more elements to the head of a list that is stored in a specified key.                                                                                          
+          
+        - **redisKey** : The name of the key where the list is stored.
+        - **redisStrings** : One or more elements that you want to add to the head of the list.
+        
+        While invoking the API, the above two parameters values extracted from the response of SimpleStockQuoteService. Then insert as a input value.
+        
+        <img src="../../../../assets/img/connectors/redis-ipush-parameters.png" title="iPush parameters" width="500" alt="iPush parameters"/> 
+    
+6. To get the input values in to the `iPush` we can use the [property mediator](../../../mediators/property-Mediator). Navigate into the **Palette** pane and select the graphical mediators icons listed under **Mediators** section. Then drag and drop the `Property` mediators into the Design pane as shown bellow.
+    
+   > **Note**: That the properties should be add to the pallet before create the operation.
+        
+        The parameters available for configuring the Property mediator are as follows:
+    
+    1. Add the property mediator to capture the `symbol` value from the response of SimpleStockQuoteService. The 'symbol' contains name of the stock quote. 
+   
+        - **name** : symbol
+        - **value expression** : $body/soapenv:Envelope/soapenv:Body/ns:getQuoteResponse/ax21:symbol
+   
+        <img src="../../../../assets/img/connectors/redis-getsymbol-properties1.png" title="Add property mediators to get symbol" width="600" alt="Add property mediators to get symbol"/>
+    
+    2. Add the property mediator to capture the `volume` values. The message contains content of the SMS message.                  
+   
+        - **name** : volume
+        - **value expression** : $body/soapenv:Envelope/soapenv:Body/ns:getQuoteResponse/ax21:volume
+     
+        <img src="../../../../assets/img/connectors/redis-getvolume-properties1.png" title="Add property mediators to get volume" width="600" alt="Add property mediators to get volume"/>  
+        
+7. Get a response to the user.
```

Comment:
Forward the backend response to the API caller

Reply:
Updated.